.buildkite/pipeline.yml

```diff
@@ -0,0 +1,52 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_GVM_VERSION: 'v0.5.0' # https://github.com/andrewkroh/gvm/issues/44#issuecomment-1013231151
+  SETUP_MAGE_VERSION: "latest"
+  GH_CLI_VERSION: "2.29.0"
```

Comment:
Do we use gh cli here ? IIRC it is not, and I could be removed.

Reply:
removed